tools/mutator-utils/mutator_helper.cpp

```diff
@@ -1131,4 +1133,53 @@ void ResizeIntegerHelper::debug() {
   llvm::errs() << "integer resized\n";
   mutator->iitInTmp->getParent()->print(llvm::errs());
   llvm::errs() << "\n";
-}
\ No newline at end of file
+}
+
+void UnaryInstHelper::init() {
```

Comment:
do you need a separate init() method? it would be better to do this in the constructor, or in just a regular static initializer